samples/Mvc.Client/Startup.cs

```diff
@@ -49,6 +49,11 @@ public class Startup {
                 options.ClientSecret = "98f1bf028608901e9df91d64ee61536fe562064b";
             });
 
+            app.UseLinkedInAuthentication(options => {
```

Comment:
Yeah sorry, I delete it. I can recreate one for you to test if you want to. Let me know.